ui/src/pages/namespace/Events/History/EventsList.tsx

```diff
@@ -109,29 +126,56 @@ const EventsList = ({
                   )}
                 </EventsTable>
               </Card>
-              {totalPages > 1 && (
-                <Pagination>
-                  <PaginationLink
-                    data-testid="pagination-btn-left"
-                    icon="left"
-                    onClick={() => goToPreviousPage()}
-                  />
-                  {pagesList.map((page) => (
-                    <PaginationLink
-                      active={currentPage === page}
-                      key={`${page}`}
-                      onClick={() => goToPage(page)}
+              <div className="flex items-center justify-end">
```

Comment:
When we add `gap-2` to the `className` of the div `<div className="flex items-center justify-end">`, we can get rid of the extra div around the select: This would make the markup a bit cleaner